jupyterhub/handlers/base.py

```diff
@@ -978,14 +979,21 @@ def _fail_api_request(self, user):
                 port = 443 if host_info.scheme == 'https' else 80
             if port != Server.from_url(self.proxy.public_url).connect_port and port == self.hub.connect_port:
                 self.log.warning("""
-                    Detected possible direct connection to Hub's private ip: %s, bypassing proxy.
-                    This will result in a redirect loop.
-                    Make sure to connect to the proxied public URL %s
-                    """, self.request.full_url(), self.proxy.public_url)
-
+                Detected possible direct connection to Hub's private ip: %s, bypassing proxy.
```

Comment:
Looks like this got dedented by accident

Reply:
You're right.